### ISSUE TITLE
Onboarding commit message ignores lowercase char

### ISSUE THREAD

**Issue by rarkins** [renovatebot/renovate#22346]
### Discussed in https://github.com/renovatebot/renovate/discussions/22342

<div type='discussions-op-text'>

<sup>Originally posted by **simone-dandreta** May 22, 2023</sup>
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.0.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

My default.json file is:
`{
     "extends": ["config:base", ":dependencyDashboard"],
     "enabledManagers": ["dockerfile"],
     "commitMessageAction": "bump",
     "onboardingCommitMessage": "trivial: Configure Renovate"
}`

but when I look at the onboarding commit message I see:
`Trivial: Configure Renovate` with capital 'T'

and this breaks our CI checks on semantic commit messages. 
Am I missing something?

### What you tried so far.

You can look at the default.json file posted above.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>
</div>

**Comment by simone-dandreta** [comment]
I guess the problem might be [here](https://github.com/renovatebot/renovate/blob/947babe092ddf8b70357791059296eb876ed4623/lib/workers/repository/model/commit-message.ts#L80)?

**Comment by simone-dandreta** [comment]
@rarkins is there any update about this? Thank you.

**Comment by rarkins** [comment]
No. PRs welcome

**Comment by zharinov** [comment]
- Do we still need it after #25666?

**Comment by gjrtimmer** [comment]
@zharinov Yes, still needed; currently implementing Renovate Bot on GitLab (Private) Renovate:37.74.
The first letter of Onboarding PR is still a Capital case which breaks semantic commit messages and also breaks our tooling and scripting which generates CHANGELOG etc.
